nbd.opam

```diff
@@ -7,11 +7,11 @@ doc: "https://xapi-project.github.io/nbd/nbd/index.html"
 dev-repo: "https://github.com/xapi-project/nbd.git"
 bug-reports: "https://github.com/xapi-project/nbd/issues"
 
-build: ["jbuilder" "build" "-p" name "-j" jobs]
-build-test: ["jbuilder" "runtest" "-p" name]
+build: ["dune" "build" "-p" name "-j" jobs]
+build-test: ["dune" "runtest" "-p" name]
 
 depends: [
-  "jbuilder" {build & >= "1.0+beta10"}
+  "dune" {build & >= "1.0+beta10"}
```

Comment:
I think you can drop the version constraint, dune never had a 1.0+beta10 version

Reply:
Thanks, done